lab-edward-smith/readme.MD

```diff
@@ -0,0 +1,64 @@
+# Cowsay As A Service
+
+To use CaaS:
+
+```sh
+$ npm install
+```
+Usage
+
+```sh
+$ node server.js
```

Comment:
This command doesn't work because your server.js file is in the 'lib' directory. It really should be in your project root instead.